src/components/DropZone/components/FileUpload/FileUpload.tsx

```diff
@@ -114,22 +96,25 @@ export function FileUpload(props: FileUploadProps) {
   const smallView =
     size === 'small' ? (
       <Stack vertical spacing="tight">
-        <Icon source={DragDropMajorMonotone} color="inkLightest" />
+        <img width="20" src={uploadArrow} alt="" />
```

Comment:
I think we can remove `<Stack />` for small sizes. Seems to be causing some issues and will make it harder to vertically align